styles.css

```diff
@@ -0,0 +1,16 @@
+body{
+    background-color: blue;
+    font-family: 'Poppins', sans-serif;
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.4;
+  color: var(--color-white);
+  margin: 0;
```

Comment:
Please make all properties to be on the same line for making your code to be clean.